../gegl-0.4.30/operations/external/png-load.c:85:9: warning:
implicit declaration of function 'strptime' is invalid in C99 [-Wimplicit-function-declaration]

Index: operations/external/png-load.c
--- operations/external/png-load.c.orig
+++ operations/external/png-load.c
@@ -25,7 +25,7 @@
 #endif
 
 #ifdef HAVE_STRPTIME
-#define _XOPEN_SOURCE
+#define _XOPEN_SOURCE 700
 #include <time.h>
 #endif
 
